thunder/__init__.py

```diff
@@ -203,8 +203,18 @@ def __init__(self, model, compiled_model_call):
         super().__init__()
         self._model = model
 
+        del self.training
```

Comment:
Why is this here? A comment would be helpful

Reply:
Indeed!